web/.prettierignore

```diff
@@ -5,5 +5,8 @@
 yarn.lock
 
 world.json
+world-aggregated.json
+excluded-aggregated-exchanges.json
```

Comment:
nitpick but maybe `exchanges-excluding-aggregated` is easier to understand and fits better with the rest of our naming conventions.

Reply:
Hmm, it's a list of exchanges to exclude when showing the aggregated view, maybe ````exchanges-to-exclude-from-aggregated-view``` but it's very long.